packages/ui/Routable/Link.js

```diff
@@ -7,11 +7,70 @@ import {Component} from 'react';
 
 import useLink from './useLink';
 
+/**
+ * A component that is used to make a trigger to navigate inside of {@link ui/Routable.Routable} component.
```

Comment:
Does the below sentence make sense?
```suggestion
 * A component that is used to make a link to navigate among {@link ui/Routable.Route} components.
```

Reply:
It is more understandable. 👍